api/src/main/java/org/openmrs/module/pihmalawi/sql/MysqlScriptRunner.java

```diff
@@ -0,0 +1,207 @@
+package org.openmrs.module.pihmalawi.sql;
+
+/*
+ * Slightly modified version of ScriptRunner
+ * https://github.com/BenoitDuffez/ScriptRunner
+ */
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.LineNumberReader;
+import java.io.PrintWriter;
+import java.io.Reader;
+import java.io.UnsupportedEncodingException;
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.ResultSetMetaData;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class MysqlScriptRunner {
```

Comment:
I'd probably rename all of these now, since these are no longer mysql-specific.  Before, they were, as we were running mysql command directly.  Now, presumably, these are DB-agnostic (or at least they are only as DB-specific as the scripts that you run).